test/jb_funding_cycle_store/configure_for.test.js

```diff
@@ -0,0 +1,1396 @@
+import { expect } from 'chai';
```

Comment:
can you run `yarn pretty`? I'm noticing a lot of formatting things seem to be off. long lines are harder to review.

Reply:
word